src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderNoPackage.java

```diff
@@ -4,6 +4,15 @@
 
 import java.util.*;
```

Comment:
please do me a favor and change `non-compiled with javac: contains specially crafted set of imports for testing` to `non-compiled with javac: contains no package for testing`

Reply:
Done. 😄